### ISSUE TITLE
my game keeps crashing/better optimization

### ISSUE THREAD

**Issue by speedgamer3000** [luckydog7/Funkin-android#26]
help, my game keeps crashing when i choose a song to play if i already played 1 song before. so i can only play 1 song before it crashes. pls fix! also, its really laggy, so can you optimize it on my device: oppo a83. thank you!

**Comment by yassir1186** [comment]
Like me i have so crashes in the game

**Comment by yassir1186** [comment]
Really like me i'm so much have crash in the game and that make the game so much bored my device is (huawei prime Y7)

**Comment by baconmanyt** [comment]
If I click a week it just crashes my device is redmi

**Comment by aminos-ze-best** [comment]
Yeah, it crashes and lags for me too. My device is Xiaomi Redmi 9A.

**Comment by kalebplayz125** [comment]
I can't go into weeks songs or whatever 
My device is a Samusng galaxy A10e

**Comment by matthew99729** [comment]
The same thing happens to me. I even compressed every song and texture in the game and successfully reduced the game's size by  50% because I thought it had something to do with android's memory limit being reached but compressing the game files changed nothing.

**Comment by kalebplayz125** [comment]
> The same thing happens to me. I even compressed every song and texture in the game and successfully reduced the game's size by 50% because I thought it had something to do with android's memory limit being reached but compressing the game files changed nothing.

i dont know why
